contracts/pcv/balancer/BPTLens.sol

```diff
@@ -72,7 +76,12 @@ contract BPTLens is IPCVDepositBalances {
     }
 
     function balance() public view override returns (uint256) {
-        (IERC20[] memory _tokens, uint256[] memory balances, ) = VAULT.getPoolTokens(id);
+        (
```

Comment:
This seems to change something significant - in the previous version, tokens and balances were assigned from indexes 0 and 1 from the result tuple, and in the new version, they're assigned from indexes 1 and 2 - am I reading this right?

Reply:
😨 yes

Reply:
prettier formatted it in a **very** weird way but I think we're good, the code change is :  (x, y, ) = ... -> ( , y, )

Reply:
Ahh yep I see now. Sounds good!